com.ibm.wala.cast.js.nodejs/build.gradle

```diff
@@ -50,3 +48,10 @@ tasks.named('processResources') {
 tasks.named('clean') {
 	dependsOn 'cleanUnpackNodeJSLib'
 }
+
+tasks.named('test') {
+	maxHeapSize = '800M'
+
+	// fails with java.lang.OutOfMemoryError for unknown reasons
+	exclude '**/NodejsRequireTargetSelectorResolveTest.class'
```

Comment:
Is this a new failure?

Reply:
No, this has been failing since at least April 2018, according to 0e629f68e7955fc72605680b1ef8121c015ee0c6.

https://github.com/wala/WALA/blob/d93b0217bd80aae5a7b2a49a393db6655ee5daf5/com.ibm.wala.cast.js.nodejs.test/build.gradle#L16-L17